Makefile

```diff
@@ -46,8 +46,15 @@ $(BUILD_DIR)/linux-amd64/$(BINARY_NAME):
 $(BUILD_DIR)/windows-amd64/$(BINARY_NAME).exe:
 	CGO_ENABLED=0 GOARCH=amd64 GOOS=windows go build -ldflags="$(LDFLAGS)" -o $@ $(GO_BUILDFLAGS) ./cmd/admin-helper/
 
+$(BUILD_DIR)/macos-universal/$(BINARY_NAME): $(BUILD_DIR)/macos-amd64/$(BINARY_NAME) $(BUILD_DIR)/macos-arm64/$(BINARY_NAME)
+	mkdir -p $(BUILD_DIR)/macos-universal
+	cd $(BUILD_DIR) && lipo -create macos-amd64/$(BINARY_NAME) macos-arm64/$(BINARY_NAME) -output macos-universal/$(BINARY_NAME)
+
 .PHONY: cross ## Cross compiles all binaries
-cross: $(BUILD_DIR)/macos-amd64/$(BINARY_NAME) $(BUILD_DIR)/macos-arm64/$(BINARY_NAME)-arm64 $(BUILD_DIR)/linux-amd64/$(BINARY_NAME) $(BUILD_DIR)/windows-amd64/$(BINARY_NAME).exe
+cross: $(BUILD_DIR)/macos-amd64/$(BINARY_NAME) $(BUILD_DIR)/macos-arm64/$(BINARY_NAME) $(BUILD_DIR)/linux-amd64/$(BINARY_NAME) $(BUILD_DIR)/windows-amd64/$(BINARY_NAME).exe
+
+.PHONY: macos-universal ## Creats macOS universal fat binary
+macos-universal: clean lint test $(BUILD_DIR)/macos-universal/$(BINARY_NAME)
 
 .PHONY: release
 release: clean lint test cross
```

Comment:
Would be nicer if we could add `macos-universal` here, and copy it to the release dir as well. But I understand for now it's a bit complicated as macos-universal can only be used on a macos build machine.

Reply:
after using `makefat`. its easy replace `macos-amd64` with `macos-universal` and it'll have the universal binary in the release archive